getting-started/dc2-guide.rst

```diff
@@ -0,0 +1,190 @@
+* One set of reference catalogs are called ``cal_ref_cat_2_2``, and have some filtermap definitions that must be specified
+* On NCSA's ``lsstdevl`` in ``/repo/dc2``, tracts 3828 and 3829 are in the collection ``2.2i/defaults/test-med-1``
+* A smaller CI dataset, just patch 24 in tract 3828, is in ``2.2i/defaults/ci_imsim`` and corresponds to the GitHub repo ``testdata_ci_imsim``
+
+To see some collections, try, e.g.,
+
+.. prompt:: bash
+
+   butler query-collections /repo/dc2 --chains=tree "2.2i/*"
+   butler query-collections /repo/dc2 --chains=tree "u/mrawls/*"
+
+These commands should print a list of collections that meet the search criteria.
+The "tree chain" view is a handy way to see the "chained" data products that are included with each collection, such as calibs, skymaps, refcats, etc.
+
+To see some available datasets for processing, try, e.g.,
+
+.. prompt:: bash
+
+   butler query-data-ids /repo/dc2 tract patch visit --collections='2.2i/defaults/test-med-1' --where "skymap='DC2' AND band='g' AND tract=3828 AND patch=47 AND instrument='LSSTCam-imSim'"
+
+This command should print a list of data IDs that meet the search criteria, along with their tract, patch, and visit number.
+Certain arguments are required after the ``--where``, including ``skymap`` and ``instrument``, while most others are optional.
+
+Processing Data
+===============
+
+Now it's time to process some data.
+In this guide, we will start with raws, run "processCcd" (which includes :py:mod:`lsst.ip.isr.IsrTask`, :py:mod:`lsst.pipe.tasks.characterizeImage.CharacterizeImageTask`, and :py:mod:`lsst.pipe.tasks.calibrate.CalibrateTask`), and make good seeing coadd templates.
+
+In a second pipeline, we will run difference imaging using the templates we just built and save the results in an Alert Production Database (APDB).
+
+Building good seeing templates
+------------------------------
+
+Here is the pipeline we will use.
+Note that multiple python configuration options can be used by typing a placeholder continuation character (e.g., ``|``) followed by one python config declaration per line.
+
+.. prompt:: yaml
+
+   description: An AP pipeline for building templates with LsstCam-imSim data
+
+   instrument: lsst.obs.lsst.LsstCamImSim
+   imports:
+   - location: $AP_PIPE_DIR/pipelines/ApTemplate.yaml
+   tasks:
+     calibrate:
+       class: lsst.pipe.tasks.calibrate.CalibrateTask
+       config:
+         connections.astromRefCat: 'cal_ref_cat_2_2'
+         connections.photoRefCat: 'cal_ref_cat_2_2'
+         astromRefObjLoader.ref_dataset_name: 'cal_ref_cat_2_2'
+         photoRefObjLoader.ref_dataset_name: 'cal_ref_cat_2_2'
+         python: |
+         config.astromRefObjLoader.filterMap = {band: 'lsst_%s_smeared' % (band) for band in 'ugrizy'};
+         config.photoRefObjLoader.filterMap = {band: 'lsst_%s_smeared' % (band) for band in 'ugrizy'};
```

Comment:
I think this is missing one level of indentation; I get a syntax error as-is. The block under the pipe needs to be indented beyond the depth of the `python` node.